<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_lens">
       <item>
        <property name="text">
         <string>Objective</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Condenser</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Scan X</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Scan Y</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Vmax(V)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_Vmax">
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="value">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="pushButton_add_power_supply">
       <property name="text">
        <string>Add Power Supply</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Imin(A)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="pushButton_Finish">
       <property name="text">
        <string>Finish</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_Vmin"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Vmin(V)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_Imax">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="value">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Lens</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Imax(A)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Power Supply Id:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_Imin">
       <property name="decimals">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_Id">
       <item>
        <property name="text">
         <string>GPP-1326</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GPP-2323 #1 (Channel1)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GPP-2323 #1 (Channel2)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GPP-2323 #2 (Channel1)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GPP-2323 #2 (Channel2)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="pushButton_preview">
       <property name="text">
        <string>Check  Power Supply List</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="pushButton_delete">
       <property name="text">
        <string>Delete Power Supply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
